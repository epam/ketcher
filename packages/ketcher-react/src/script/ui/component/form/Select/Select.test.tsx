import { render } from '@testing-library/react'

import Select from './Select'

const mockProps = {
  options: [
    { value: 'option1', label: 'option1' },
    { value: 'option2', label: 'option2' },
    { value: 'option3', label: 'option3' }
  ],
  onChange: jest.fn()
}

describe('Select component should be rendered correctly', () => {
  it('should be rendered without crashing', () => {
    const { asFragment } = render(<Select {...mockProps} />)
    expect(asFragment).toMatchSnapshot()
  })
})
