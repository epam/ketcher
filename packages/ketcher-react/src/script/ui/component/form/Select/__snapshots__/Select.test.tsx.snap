// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select component should be rendered correctly should be rendered without crashing 1`] = `[Function]`;
